web/source/osk/visualKeyboard.ts

```diff
@@ -1871,7 +1915,7 @@ namespace com.keyman.osk {
       var b: HTMLElement = _Box, bs=b.style;
       bs.height=bs.maxHeight=oskHeight+'px';
 
-      b = <HTMLElement> b.childNodes.item(1).firstChild;
+      b = this.kbdDiv;
```

Comment:
This is literally the exact same object the original line was trying to reference.  That said, sometimes the old version accidentally accessed the predictive banner instead... so this is actually an improvement / fix for other aspects, too.

Reply:
After finding that this PR caused #2834, it turns out that this line is the culprit.  It's not `this.kbdDiv`, but instead `this.kbdDiv.firstChild` that the original line would have referred to.